From 8232a0ff48ce0959e20db9ffa1c4e16d4657dbec Mon Sep 17 00:00:00 2001
Message-Id: <patch-v4-13.80-8232a0ff48c-20210804T164957Z-avarab@gmail.com>
In-Reply-To: <cover-v4-00.80-00000000000-20210804T164956Z-avarab@gmail.com>
References: <cover-00.12-00000000000-20210721T233307Z-avarab@gmail.com>
	<cover-v4-00.80-00000000000-20210804T164956Z-avarab@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Thu, 8 Jul 2021 19:27:22 -0700
Subject: [PATCH v4 13/80] pkt-line: replace "stateless separator" with
 "response end"

In 0181b600a6 (pkt-line: define PACKET_READ_RESPONSE_END, 2020-05-19),
the Response End packet was defined for Git's network protocol. When the
patch was sent, it included an oversight where the error messages
referenced "stateless separator", the work-in-progress name, over
"response end", the final name chosen.

Correct these error messages by having them correctly reference
a "response end" packet.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
Signed-off-by: Junio C Hamano <gitster@pobox.com>
---
 pkt-line.c    | 2 +-
 remote-curl.c | 2 +-
 serve.c       | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/pkt-line.c b/pkt-line.c
index 8f9bc68ee28..4f0264b4793 100644
--- a/pkt-line.c
+++ b/pkt-line.c
@@ -103,7 +103,7 @@ void packet_response_end(int fd)
 {
 	packet_trace("0002", 4, 1);
 	if (write_in_full(fd, "0002", 4) < 0)
-		die_errno(_("unable to write stateless separator packet"));
+		die_errno(_("unable to write response end packet"));
 }
 
 int packet_flush_gently(int fd)
diff --git a/remote-curl.c b/remote-curl.c
index 75532a8baea..3c6eceb7be6 100644
--- a/remote-curl.c
+++ b/remote-curl.c
@@ -602,7 +602,7 @@ static int rpc_read_from_out(struct rpc_state *rpc, int options,
 			memcpy(buf - 4, "0000", 4);
 			break;
 		case PACKET_READ_RESPONSE_END:
-			die(_("remote server sent stateless separator"));
+			die(_("remote server sent unexpected response end packet"));
 		}
 	}
 
diff --git a/serve.c b/serve.c
index c046926ba14..4bf5e9b0c16 100644
--- a/serve.c
+++ b/serve.c
@@ -218,7 +218,7 @@ static int process_request(void)
 			state = PROCESS_REQUEST_DONE;
 			break;
 		case PACKET_READ_RESPONSE_END:
-			BUG("unexpected stateless separator packet");
+			BUG("unexpected response end packet");
 		}
 	}
 
-- 
2.33.0.rc0.587.gbfc8ffe61b6

